CHANGELOG/v1.7.0-rc.1.md

```diff
@@ -0,0 +1,455 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+
+* Enable kubeadm feature gates mutation
+* Add public function to create new CRD migrator
+* Add MachineSetReady condition to MachineDeployment
```

Comment:
I would add "Set MachinePool feature flag default to true + Beta" as a highlight. It merged since the last beta release.

#10141

Reply:
Sounds good to me! Thanks for the suggestion.